src/components/common/ErrorsDisplay.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { array } from 'prop-types';
+
+const DisplayErrors = ({ errors }) => {
```

Comment:
Just `Errors` or `FormErrors` or something as a noun is OK. No verb needed, it's kind of obvious you want to display something if you write a component.

Also, it's common practice to name the files the same as the component they host.